rotkehlchen/globaldb/updates.py

```diff
@@ -477,8 +627,8 @@ def _perform_update(
                 version += 1
                 continue
 
+            url = f'https://raw.githubusercontent.com/rotki/assets/{self.branch}/updates/{version}/updates.sql'  # noqa: E501
```

Comment:
Hm, maybe move it to a constant, make something like

`'https://raw.githubusercontent.com/rotki/assets/{branch}/updates/{version}/updates.sql'`

and here just format?